project-ui/Dockerfile

```diff
@@ -1,6 +1,12 @@
-FROM node:15.13-alpine
+FROM balenalib/raspberrypi4-64-alpine-node
 WORKDIR /project-ui
-ENV PATH="./node_modules/.bin:$PATH"
-COPY . .
-RUN npm install
+
+# install node dependencies
+COPY package.json package-lock.json ./
+RUN npm i
+
+# run app (dev mode)
```

Comment:
I'm sure you're aware of this, but you're current code runs react in development mode. Obviously not an issue for this lab project, but thought I'd make a note of it.